apps/vite/src/components/calendar/Panchang.tsx

```
@@ -10,70 +20,110 @@ const PanchangTableRow = ({ label, value }: { label: string; value: string }) =>
   children,
-  bgColor,
+  classes,
 }: {
   title: string
   children: React.ReactNode
-  bgColor: string
+  classes?: string
 }) => {
   return (
-    <div className={`mb-8`}>
+    <div className={`my-8`}>
```

Biome

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)